integration/helpers.go

```diff
@@ -152,3 +153,14 @@ func BuildBaseEvent(namespace string) eventtypes.Event {
 		CommonData: BuildCommonData(namespace),
 	}
 }
+
+func GetTestPodIP(ns string, podname string) string {
+	cmd := exec.Command("kubectl", "-n", ns, "get", "pod", podname, "-o", "jsonpath='{.status.podIP}'")
+	r, err := cmd.Output()
+	if err != nil {
+		return ""
+	}
+
+	ip := string(r)
+	return ip[1 : len(ip)-1]
+}
```

Comment:
Feel free to squash this with my previous commit. I only wanted to show that as an example, so there is not need for my commit to be there.